src/ProductConstructionService/DependencyUpdaterJob/DependencyUpdaterJob.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can we maybe drop the `Job` suffix from the name? This will be an exe that updates dependencies but the fact we deploy it as a job is a happenstance and we could decide to run this differently too

Reply:
The reason I added it is because it only makes sense to run as a job. Right now, `DependencyUpdater` is a self contained service that you kind of run and forget about. If you just run the `DependencyUpdater` it will trigger once, and die, so I think having the `Job` suffix there indicates that it only runs once

Reply:
Well, actually, maybe the whole name is off. It should be something about triggering subscriptions automatically because it does not actually update anything

Reply:
SubscriptionCron or something like that makes more sense

Reply:
that's a fair point. What about ScheduledSubscriptionTriggerer? Or ScheduledSubscriptionActivator

Reply:
renamed, as we agreed